src/docs/metrics/community-messages-count/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Community Messages Count
+author: Ivan
+date: 2020-04-13
+# REF metrics-hub/metricshub/community_messages.py
+---
+
+## Definition
+
+Community Messages Count is build on top of the [Social Data](/metrics/details/social-data).
+
+Community Messages Count returns the number of messages written in the
+asset's own community chat (telegram server).
```

Comment:
let's duplicate it in yaml description?

Reply:
Let's have a separate PR dedicated to adding short descriptions to all metrics